test/dlc_tests/benchmark/ec2/pytorch/training/test_performance_pytorch_training.py

```diff
@@ -97,7 +97,7 @@ def execute_pytorch_gpu_py3_imagenet_ec2_training_performance_test(
 
 
 def post_process_pytorch_gpu_py3_synthetic_ec2_training_performance(connection, log_location):
-    last_lines = connection.run(f"tail {log_location}").stdout.split("\n")
+    last_lines = connection.run(f"tail -n 20 {log_location}").stdout.split("\n")
```

Comment:
how many lines does this test produce? is last 20 enough?

Reply:
yes, 20 lines are sufficient for getting the required numbers.